back/src/config/pg/kysely/model/database.ts

```diff
@@ -359,7 +359,8 @@ interface Users {
   last_name: string;
   created_at: Generated<string>;
   updated_at: Generated<Timestamp>;
-  external_id: string | null;
+  external_id_inclusion_connect: string | null;
+  external_id_pro_connect: string | null;
```

Comment:
vous preferez pas virer le external ? :

```suggestion
  inclusion_connect_sub: string | null;
  pro_connect_sub: string | null;
```